Add origami-version GitHub Action

JakeChampion opened the pull request.

origamiserviceuser commented:
o-stepped-progress bundle size difference from 2.0.0 to 8c4789e01046055a943d6bdec09a107590596677
No significant bundle size differences found.

JakeChampion committed:
Add origami-version GitHub Action

When this is merged, we can have automated releases by using the release-type labels "major"/"minor"/"patch" on a pull-request before merging the pull-request.

origamiserviceuser commented:
o-stepped-progress bundle size difference from 2.0.0 to 9e6780c71a2967f75a3dcd3384f3da79d1bfc882
No significant bundle size differences found.

JakeChampion commented:
This helps achieve the OKR outlined here https://github.com/Financial-Times/origami/issues/39
